src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeff.java

```diff
@@ -40,29 +36,30 @@
  * </ul>
  *
  */
-@DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Likelihood-based test for the consanguinity among samples (InbreedingCoeff)")
```

Comment:
It this not supposed to be documented ?

Reply:
Whoops, that looks like I missed something in the merge conflicts